### ISSUE TITLE
examples do not work

### ISSUE THREAD

**Issue by hq9000** [vega/vega-webgl-renderer#6]
hi! I tried this one out and it didn't work, unfortunately. I went on to look at examples linked from the readme and they didn't work either, with a similar error:

![image](https://github.com/user-attachments/assets/023282b8-7f05-4d59-815b-99fa6f018747)

my guess is that something got broken as of vega 5.30. On 5.0.0 it seems to work.

**Comment by jeffbaumes** [comment]
Indeed. It seems that Renderer is now a real JS class that should be subclassed from, instead of a constructor object. I don't know how involved a fix would be to make it work with recent vega.
